ansible/roles/linux-ansible-init/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+ansible_init_venv_path: "/home/ipuuser/tools"
```

Comment:
Make this something a bit less specific to your use case, e.g. `/opt/ansible`.

Reply:
Done